docs/source/getting_started/installation.rst

```diff
@@ -224,23 +225,29 @@ To check the project works, open a terminal in the project root directory,
 which is the ``cs-field-guide/`` directory (should contain a file called
 ``csfg``).
 
-Type the following command into the terminal (we will cover this command
-in more detail on the next page):
+Type the following command into the terminal (we cover this command
+in more detail on the helper commands page):
 
 .. code-block:: bash
 
     $ ./csfg start
 
-If this is the first time you're running this script, it will need to build
-system images.
-This can take some time, roughly 15 to 30 minutes, depending on your computer
-and internet speed (we recommend grabbing a cup of tea and watching an episode
-of Brooklyn Nine-Nine on Netflix).
+If this is the first time you're running this script it will need to build system 
+images. 
+This may take roughly 15 to 30 minutes, depending on your computer and internet speed.
```

Comment:
This time is mostly for `./csfg build` which is covered elsewhere. You can test how long this actually takes by removing everything related to Docker (method in #906) and then running the commands prefixed with `time`, i.e. `time ./csfg start`

Reply:
I think it's probably still useful to note here? The installation instructions don't state to run `./csfg build` before `./csfg start` because the start command (basically) runs the `./csfg build` command automatically the first time. Which is why it will take some time.

It doesn't call `./csfg build` but it does the same commands.

I dunno, I can put `./csfg build` in the installation part if you like?

Reply:
It does run build the first time, I was mistaken on that. Okay this is fine then